### DISCUSSION TITLE
Some potential future coverage on Youtube

### DISCUSSION THREAD

**Discussion by eervin123** [kernc/backtesting.py#216] [General]
If you haven't already seen [PartTimeLarry's](https://www.youtube.com/channel/UCY2ifv8iH1Dsgjrz-h3lWLQ) youtube channel, it is worth checking out.  He builds some great "python for finance" projects and has a nice teaching style for beginners. 

I mention it here because he is likely to start doing some videos with backtesting.py in there. Hopefully getting some more well deserved eyeballs on this project. In the past he was primarily working with backtrader, but I asked him to take a look at backtesting.py and it [appears](https://youtu.be/uiPMCJ0MEbM?t=1541) he is going to incorporate it into some of his future vids.   He's very kind and helpful, like you @kernc and I think his followers would really appreciate the simplicity of backtesting.py over backtrader.  I know I do.

Github profile is here.  https://github.com/hackingthemarkets

**Comment by kernc** [comment]
I have Google Alerts set up, but that service is worth less than it costs. Always appreciate your kind words, and thanks for keeping me posted. :heart: 

I've subbed with notifications, but might still miss our featured review if not explicitly mentioned as it's unlikely I'll be watching whole episodes through. Welcome a further heads-up if you notice that happening. :smile:

**Reply by eervin123** [reply]
Will do!
